main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        system
+//  action 1:    the system upgrades
+system.upgrade();
+//  action 2:    the system downgrades
+system.downgrade();
+//  action 3:    the system collects error messages
+system.collectErrorMessage();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        dog
+//  action 1:    the dog bites
+dog.bite();
+//  action 2:    the dog runs
+dog.run();
+//  action 3:    the dog barks
+dog.bark();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        plane
+//  action 1:    the plane takes off
+plane.takeOff();
+//  action 2:    the plane lands
+plane.land();
+//  action 3:    the plane flys
+plane.fly();
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        ball
+//  action 1:    the ball bounces
+ball.bounce();
+//  action 2:    the ball rolls down
+ball.rollDown();
+//  action 3:    the ball floats on water
+ball.floatOnWater();
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:         video
+//  action 1:     the video plays
+video.play();
+//  action 2:     the video pauses
+video.pause();
+//  action 3:     the video repeats n times
+video.repeat(n);
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:         image
+//  action 1:     the image zooms in
+image.zoomIn();
+//  action 2:     the image zooms out
+image.zoomOut();
+//  action 3:     the image resizes
+image.resize();
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:         plugin
+//  action 1:     the plugin activates
+plugin.activate();
+//  action 2:     the plugin deactivates
+plugin.deactivate();
+//  action 3:     the plugin installs
+plugin.install();
 //  8)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:         player
+//  action 1:     the player dribbles
+player.dribble();
+//  action 2:     the player passes the ball
+player.pass();
+//  action 3:     the player shots
+player.shot();

```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.